cmake/tryCompileTestBCrypt.cpp

```diff
@@ -0,0 +1,9 @@
+#include <Windows.h>
+#include <bcrypt.h>
+#include <cstdio>
+#pragma comment(lib, "bcrypt.lib")
```

Comment:
This will not work properly when compiling for Windows using MINGW.

Try adding `LINK_LIBRARIES bcrypt` to the `try_compile` call in `CMakeLists.txt`.

Reply:
done